docs/dapp/tools/remix.md

```diff
@@ -0,0 +1,126 @@
+---
+description: Remix web IDE
+---
+
+# Remix
+
+## Intro
+
+[Remix] is a web-based Integrated Development Environment (IDE) designed for
+developing, testing, and deploying smart contracts on the Ethereum Network.
+This guide will show you how to use Remix in conjunction with MetaMask on the
+Sapphire Network.
+
+:::info
+
+For comprehensive details about Remix's features, consult the
+[Remix documentation].
+
+:::
+
+## Prerequisites
+
+1. Install the [MetaMask browser extension][metamask]
+2. Configure your networks:
+   - Add Sapphire networks to MetaMask using the `Add to MetaMask` buttons on
+  our [network page]
+   - (Optional) Configure local network settings if you're using the Sapphire
```

Comment:
(not relevant) to this PR, but maybe that could be a convenient local dev button to have